sources/packages/web/src/views/student/application-offering-change-request/RequestedApplicationOfferingDetails.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <body-header-container>
+    <template #header>
+      <body-header :title="changeRequest.studentFullName">
+        <template #subtitle
+          ><detail-header :headerMap="headerDetailsData"
+        /></template>
+        <template #status-chip>
+          <status-chip-application-offering-change
+            v-if="changeRequest.status"
+            :status="changeRequest.status"
+          />
+        </template>
+      </body-header>
+    </template>
+    <hr class="horizontal-divider" />
+    <h2 class="category-header-large primary-color">Application Details</h2>
+    <p v-if="changeRequestNotApproved">
+      Below displays the requested changes from your institution. You can
+      compare your active application details by switching tabs located above.
+    </p>
+    <offering-view :offeringId="changeRequest.requestedOfferingId" />
+    <hr class="horizontal-divider" />
+    <h2 class="category-header-large primary-color">Request details</h2>
+    <p v-if="changeRequestNotApproved">
+      Contact your institution for more information if there are no differences
+      in the requested change above. There may be proposed changes that aren't
+      displayed such as study costs.
+    </p>
+    <v-textarea
```

Comment:
should this be a text-area, IMO, `content-group-info` is more aligned with figma
![image](https://github.com/bcgov/SIMS/assets/77353155/079ddf7f-05de-41e5-8346-2deccc2865c0)